orisa/orisa.py

```diff
@@ -939,7 +939,7 @@ async def register(self, ctx, type: str = "pc", *, online_id: str = None):
                 # Translators: Check how Discord translates "Allow direct messages from server members" in your local language and use the same term
                 _(
                     "I'm not allowed to send you a DM. Please right click on the Discord server, "
-                    'select Privacy Settings, and enable "Allow direct messages from server members." Then try again.'
```

Comment:
PEP8 recommends avoiding blackslash escapes by switching the string quoting when necessary, this is also what black does.

Reply:
True https://www.python.org/dev/peps/pep-0008/#string-quotes It messes with my head instead of make it more readable, but will revert.

Reply:
you don't need to, black already did. Your PR is merged already, with only a few adjustments (mostly those I commented on).